### PR TITLE
Create CNAME

### PR DESCRIPTION
**Pull request by Kuan-Wei-Kuo**

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+rnawealth.andydiary.com
```
Comment (zack7498) [inline CNAME:1]: require fixed

Comment (Kuan-Wei-Kuo) [reply CNAME:1]: We need create CNAME file for github page.

Comment (zack7498) [reply CNAME:1]: OK, Approve now.

**Review by zack7498 — APPROVED** [review]
ok

**Pull request merged**